src/createServer.js

```diff
@@ -1,11 +1,186 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
 
 function createServer() {
   // Use express to create a server
   // Add a routes to the server
   // Return the server (express app)
+  const app = express();
+
+  let usersData = [];
+  let exprensesData = [];
+
```

Comment:
You can add 
`app.use(express.json());`
instead of using `express.json()` in every function